src/Curl.CommandLine.Parser/Models/Parsing/HeaderParameterEvaluator.cs

```diff
@@ -24,8 +24,8 @@ protected override void EvaluateInner(ref Span<char> commandLine, ConvertResult<
         }
 
         var headerKey = value.Slice(0, separatorIndex)
-            .Trim()
-            .ToString();
+            .ToString()
```

Comment:
Why do we need these changes?

Reply:
This is because of the [Trim method](https://learn.microsoft.com/en-us/dotnet/api/system.memoryextensions.trim?view=net-6.0#system-memoryextensions-trim-1(system-span((-0))-system-readonlyspan((-0)))) which is not available in netstandard2.1

Reply:
let's drop .netstandard2.1 then

Reply:
Done updating.